napalm_ios/ios.py

```diff
@@ -592,6 +592,13 @@ def get_optics(self):
             port_detail['physical_channels'] = {}
             port_detail['physical_channels']['channel'] = []
 
+            # If interface is shutdown it returns "N/A" as output power.
+            # Converting that to -40.0 float
+            try:
+                float(output_power)
+            except ValueError:
+                output_power = -40.0
```

Comment:
Yeah, I was thinking about this. I'm wondering whether it is good to use 0.0 as default value for optical powers as it is (in some cases) completely valid value. So 0.0 doesn't indicate not supported(available) see [Cisco 10GBASE SFP+ Modules datasheet ](http://www.cisco.com/c/en/us/products/collateral/interfaces-modules/transceiver-modules/data_sheet_c78-455693.pdf) table 2 for example. Cisco uses value of -40.0 at Rx side to indicate "no light". Nevermind, as technology evolves somebody will create more sensitive receivers and  -40.0 will be again valid value. Should we default to -100 or something?

Even you are fine with such value I think we should have it consistent across platforms and all values of Rx, Tx, instant, max, min, avg.